### ISSUE TITLE
TextField corners

### ISSUE THREAD

**Issue by Poivin** [JFormDesigner/FlatLaf#65]
Hi,

it seems that components like comboBox or buttons can have rounded corners (Component.arc) but it is not available for textFields or textComponents like TextArea. Am i wrong? 
To my mind, it could enhance the consistency between all UI components. Would it be suitable to have the "arc" custom property at textComponent level?
![Capture](https://user-images.githubusercontent.com/61098745/74597671-49e5c200-5063-11ea-9b6c-b4ed4cd829dd.PNG)

**Comment by DevCharly** [comment]
@Poivin thanks for the idea. It is implemented in commit b52250037942ca107d9a8aeafde306f7cc2e91f8 for JTextField, JFormattedTextField and JPasswordField. 

For JTextArea, JTextPane and JEditorPane it is difficult to implement because the border is painted by the JScrollPane and round edges at the JScrollPane would be painted over by the rectangular scroll view...

To enable it use:
~~~java
UIManager.put( "TextComponent.arc", 5 );
~~~

Or use the same value as for combo boxes:
~~~java
UIManager.put( "TextComponent.arc", UIManager.getInt( "Component.arc" ) );
~~~

You can use the latest snapshot to try it out:
https://github.com/JFormDesigner/FlatLaf#snapshots

**Comment by Poivin** [comment]
Thanks a lot @DevCharly !

**Comment by DevCharly** [comment]
released in 0.28
